packages/dds/map/src/mapFactory.ts

```diff
@@ -80,7 +80,7 @@ export class MapFactory implements IChannelFactory<ISharedMap> {
  * Entrypoint for {@link ISharedMap} creation.
  * @alpha
  */
-export const SharedMap = createSharedObjectKind<ISharedMap>(MapFactory);
+export const SharedMap = createSharedObjectKind<ISharedMap & ISharedObject>(MapFactory);
```

Comment:
This is staying alpha since it includes typing for both the declarative an encapsulated APIs.

Like with SharedTree, the types are stripped down to the declarative API specific subset when reexported from fluid-framework.